core/templates/components/oppia-angular-root.component.ts

```diff
@@ -61,6 +61,10 @@
  *       loading
  */
 
+// We are using eslint disable here for multilines because we have exceeded
+// max-len limit at a lot of places so it is not possible to use eslint disable
+// next line for each of them.
```

Comment:
Put quotes around "eslint disable next line"

That said, I'm confused. I think @srijanreddy98 has already solved this by adding a max-len exclusion for "import". Can you check? If so, you should be able to fix this and drop the multiline-disable.